delta/plugins/composite-views/src/main/resources/contexts/compositeview.json

```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://bluebrain.github.io/nexus/vocabulary/",
+    "resourceTypes": {
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "resourceSchemas": {
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "viewId": {
+      "@type": "@id"
+    },
+    "CompositeView": "https://bluebrain.github.io/nexus/vocabulary/CompositeView",
```

Comment:
I don't think that's necessary, since we have vocab. See [here](https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22%40vocab%22%3A%22https%3A%2F%2Fbluebrain.github.io%2Fnexus%2Fvocabulary%2F%22%7D%2C%22%40type%22%3A%22CompositeView%22%7D)